apis/v1alpha1/destination_test.go

```diff
@@ -80,6 +80,42 @@ func TestValidateDestination(t *testing.T) {
 			},
 			want: "missing field(s): ref.kind",
 		},
+		"valid DeprecatedObjectReference": {
```

Comment:
This test description is incorrect: DeprecatedObjectReference is invalid (missing `kind`)

Reply:
There are several: 

https://github.com/knative/pkg/blob/d5915547d9db02c20f3dd1a16a1f62698ea43781/apis/volatile_time_test.go#L53-L74

https://github.com/knative/pkg/blob/ea367c13422f751fac001387b654e087ef0962a2/apis/url_test.go#L89-L125

https://github.com/knative/pkg/blob/281cda84ceb316a70c2eafc5b3250a4b72c2d468/apis/condition_types_test.go#L155-L224